content/docs/architecture/resources/scope.md

```diff
@@ -0,0 +1,77 @@
++++
+title = "Scope"
+description = "Defines a set of settings that are accessible by a specific group of users."
++++
+
+A Scope is used to control access to settings in the HobbyFarm UI. For example, a setting that is only accessible by administrators would be assigned to the `admin-ui` scope.
+
+## Kubernetes Commands
+The following commands are useful for managing Scope resources in Kubernetes.
+
+```bash
+## Get a list of all Scopes
+kubectl get scopes -n hobbyfarm-system
+
+## Create a Scope from a YAML manifest
+kubectl apply -f {scopeManifest} -n hobbyfarm-system
+
+## Edit a Scope
+kubectl edit scope {scopeName} -n hobbyfarm-system
+
+## Backup a Scope to a YAML manifest
+kubectl get scope {scopeName} -n hobbyfarm-system -o yaml > {scopeManifest}
+
+## Delete a Scope
+kubectl delete scope {scopeName} -n hobbyfarm-system
+```
+
+## Example Scope Manifests
+The following shows an example of a Scope manifest in Kubernetes.
+
+```yaml
+## Public Scope
+apiVersion: hobbyfarm.io/v1
+displayName: Public
+kind: Scope
+metadata:
+  name: public
+  namespace: hobbyfarm-system
+
+---
+
+## User UI Scope
+apiVersion: hobbyfarm.io/v1
+displayName: User UI
+kind: Scope
+metadata:
+  name: user-ui
+  namespace: hobbyfarm-system
+
+---
+
+## Admin UI Scope
+apiVersion: hobbyfarm.io/v1
+displayName: Admin UI
+kind: Scope
+metadata:
+  name: admin-ui
+  namespace: hobbyfarm-system
+```
+
+## Default Scopes
+The following scopes are available in HobbyFarm by default:
+
+| Scope | Description |
+| --- | --- |
+| public | Settings which are accessible without registration. |
+| user-ui | Settings which are accessible by registered users. |
+| admin-ui | Settings which are accessible by administrators. |
```

Comment:
"gargantua" is missing here which holds settings used by gargantua like "validate access codes"

Reply:
also admin-ui is supposed to hold settings for the admin-ui

all settings are accessible by administrators (with the corresponding rbac rules)

Reply:
@jggoebel I wasn't sure if `gargantua` was supposed to be here or not, as it wasn't in the original doc but *is* included on the deployment. I'll update to add it for clarity.

I'm not sure what you mean by the second comment. Can you clarify what you'd like to see updated?

Reply:
Per our call today, I will be updating the wording here so that makes more sense.